activerecord/lib/active_record/fixtures.rb

```diff
@@ -14,7 +14,7 @@ class FixtureClassNotFound < ActiveRecord::ActiveRecordError #:nodoc:
   # \Fixtures are a way of organizing data that you want to test against; in short, sample data.
   #
   # They are stored in YAML files, one file per model, which are placed in the directory
-  # appointed by <tt>ActiveSupport::TestCase.fixture_path=(path)</tt> (this is automatically
+  # appointed by <tt>ActiveRecord::TestCase.fixture_path=(path)</tt> (this is automatically
```

Comment:
Looks like the replace on these docs were made by mistake? `ActiveRecord::TestCase` is specific for the AR test suite and not for rails apps per se.